CHANGELOG.md

```diff
@@ -4,6 +4,27 @@
 <!--- Distributed under the Boost Software License, Version 1.0. (See accompanying -->
 <!--- file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt) -->
 
+## 0.13.0 (2023-03-08)
+
+### New features
+
+- Add `static_assert` to have better error messages with senders. ([#586](https://github.com/pika-org/pika/pull/586))
+
+### Breaking changes
+
+- Rename `done_*` with `stopped_*` to conform with stdexec implementation. ([#583](https://github.com/pika-org/pika/pull/583))
+- Remove `PIKA_WITH_BACKGROUND_THREAD_COUNTERS`. ([#588](https://github.com/pika-org/pika/pull/588))
+- Update stdexec commit used. ([#597](https://github.com/pika-org/pika/pull/597))
+- Update moodycamel concurrent queue to the latest version. ([#608](https://github.com/pika-org/pika/pull/608))
+- Cleaned up modules and moved minor functionality into `detail` namespaces. ([#607](https://github.com/pika-org/pika/pull/607), [#607](https://github.com/pika-org/pika/pull/607))
```

Comment:
607 included twice? I also think you maybe meant to use the list that's further down on `fix warning about no return...`?